integration-cli/docker_cli_daemon_test.go

```diff
@@ -978,3 +978,43 @@ func (s *DockerSuite) TestHttpsInfoRogueServerCert(c *check.C) {
 		c.Fatalf("Expected err: %s, got instead: %s and output: %s", errCaUnknown, err, out)
 	}
 }
+
+func (s *DockerSuite) TestDaemonRestartWithPausedContainer(c *check.C) {
+	d := NewDaemon(c)
+	if err := d.StartWithBusybox(); err != nil {
+		c.Fatal(err)
+	}
+	defer d.Stop()
+	if out, err := d.Cmd("run", "-i", "-d", "--name", "test", "busybox", "top"); err != nil {
+		c.Fatal(err, out)
+	}
+	if out, err := d.Cmd("pause", "test"); err != nil {
+		c.Fatal(err, out)
+	}
+	if err := d.Restart(); err != nil {
+		c.Fatal(err)
+	}
+
+	errchan := make(chan error)
+	go func() {
+		out, err := d.Cmd("start", "test")
+		if err != nil {
+			errchan <- fmt.Errorf("%v:\n%s", err, out)
+		}
+		out = strings.TrimSpace(out)
+		if out != "test" {
```

Comment:
just to be more clear, I think this could be `if name := strings.TrimSpace(out); name != "test" {}`